### ISSUE TITLE
1.5.7 not on Cocoapods

### ISSUE THREAD

**Issue by matto1990** [TheAmazingAudioEngine/TheAmazingAudioEngine#251]
The tagged release 1.5.7 is not available via Cocoapods.

Thanks for the great library!

**Comment by jsonfellin** [comment]
Have you tried something like this: 

`pod 'Alamofire', :git => 'https://github.com/Alamofire/Alamofire.git', :tag => '3.1.1'`

**Comment by matto1990** [comment]
Yes. That would work. This was more of a note to the maintainer to let them know that it might have been forgotten.
